Cargo.toml

```diff
@@ -11,5 +11,5 @@ ark-ff = { version = "0.5", default-features = false }
 ark-ec = { version = "0.5", default-features = false }
 ark-poly = { version = "0.5", default-features = false }
 ark-serialize = { version = "0.5", default-features = false, features = ["derive"] }
-fflonk = { git = "https://github.com/w3f/fflonk", default-features = false }
+fflonk = { git = "https://github.com/w3f/fflonk", branch="return-result", default-features = false }
```

Comment:
I guess you can merge `fflonk` return-result branch already? So you don't need to target this branch here